.github/workflows/libcontroller_updater.yml

```diff
@@ -0,0 +1,55 @@
+name: Libcontroller Submodule Updater
+
+on:
+  schedule:
+    - cron:  '0 10 * * *'
+
+jobs:
+  automatic-submodule-sync:
+    strategy:
+      matrix:
+        include:
+          - submodule_target_branch: R2022b
```

Comment:
Shouldn't the target branch for master be R2022b-revision-1 on the libcontroller side ?

Reply:
yes indeed